views/database/edit.erb

```diff
@@ -70,21 +70,18 @@
           </td>
         </tr>
         <tr>
-          <td>Active state</td>
           <td>
-            <% if @db %>
-              <%= @db.active %>
-            <% else %>
-              N/A
-            <% end %>
+            <input type="submit" value="Update" class="btn btn-primary">
           </td>
           <td>
-            <input type="radio" name="active" value="True"> True
-            <input type="radio" name="active" value="False"> False
+          </form>
+          <form action="/dbs/<%= @db.id %>" method='post'>
+            <input type="hidden" name="_method" value="delete">
+            <input type="submit" value="Inactive" class="btn btn-primary">
```

Comment:
Very small nitpic - I think the button should be labeled "Deactivate" or "Delete"